Dockerfile

```diff
@@ -43,4 +43,4 @@ WORKDIR /meili_data
 EXPOSE  7700/tcp
 
 ENTRYPOINT ["tini", "--"]
-CMD     /bin/meilisearch
+CMD     /bin/meilisearch
```

Comment:
can you put back the new line at the end of the file? We want to unify our yml files 😇